velox/functions/sparksql/specialforms/SparkCastHooks.h

```diff
@@ -63,6 +68,12 @@ class SparkCastHooks : public exec::CastHooks {
   exec::PolicyType getPolicy() const override;
 
  private:
+  /// Casts a number to a timestamp. The number is treated as the number of
+  /// seconds since the epoch (1970-01-01 00:00:00 UTC).
+  /// Supported input types: integer, floating-point.
```

Comment:
nit: use // for comments of private function.

> Supported input types: integer, floating-point.

Supports integer and floating-point types.

Reply:
done